Cargo.toml

```diff
@@ -1,23 +1,22 @@
 [workspace]
-members=["framework",
-        "examples/echo-reply",
-        "examples/ipv4or6",
-        "examples/macswap",
-        "examples/mtu-too-big",
-        "examples/op-errors",
-        "examples/srv6-sighup-flow",
-        # "examples/chain-test",
-        # "examples/maglev",
-        # "examples/sctp-test",
-        # "examples/tcp-reconstruction",
-        # "examples/acl-fw",
```

Comment:
via chat @drunkirishcoder said he'd remove the main fn cruft these had and bring them back in for compilation, while also adding an general example of how to add/count metrics, etc... via a tokio task (unit of logic like an erlang proc or go goroutine) from the main execution.